src/icons/src/scripts/build-icon.js

```diff
@@ -0,0 +1,41 @@
+var SVGSpriter =  require('svg-sprite');
+var fs = require('fs');
+var path = require('path');
+var mkdirp = require('mkdirp');
+
+var config = {
+  mode: {
+    symbol: {
+      // symbol mode to build the SVG
+      dest: 'dist/img/sprite', // destination foldeer
+      sprite: 'sprite.svg', // sprite name
+      example: false, // Don't build sample page
+    },
+  },
+  svg: { // General options for created SVG files
+      xmlDeclaration: false, // Add XML declaration to SVG sprite
+      doctypeDeclaration: false, // Add DOCTYPE declaration to SVG sprite
+      namespaceIDs: true, // Add namespace token to all IDs in SVG shapes
+      namespaceClassnames: true, // Add namespace token to all CSS class names in SVG shapes
+      dimensionAttributes: true // Width and height attributes on the sprite
+  }
+}
+
+var spriter = new SVGSpriter(config);
+
+// Add SVG source files
+let dir = path.join(__dirname, "../svg/")
```

Comment:
Not a big deal but if you can use `let`, use it instead of `var`, for consistency and updated style.